test/e2e/tests/ppom/ppom-blockaid-alert-simple-send.spec.js

```diff
@@ -178,11 +178,7 @@ describe('Simple Send Security Alert - Blockaid @no-mmi', function () {
       async ({ driver }) => {
         await unlockWallet(driver);
 
-        await sendScreenToConfirmScreen(
-          driver,
-          '0x985c30949c92df7a0bd42e0f3e3d539ece98db24',
-          '1',
-        );
+        await sendScreenToConfirmScreen(driver, mockMaliciousAddress, '1');
```

Comment:
for some reason, the address set previously was not the malicious address. Here I fixed it by adding the malicious address